docs/configuration.md

```diff
@@ -59,14 +59,15 @@ The `extends` property controls if your configuration should inherit from the de
 
 ### `settings: Object|undefined`
 
-The settings property controls various aspects of running Lighthouse such as CPU/network throttling and which audits should run.
+The settings property controls various aspects of running Lighthouse such as CPU/network throttling and which audits should run. You can also specify to run lighthouse against a remote browser by passing tbe chrome debugging port.
 
 #### Example
 ```js
 {
   settings: {
     onlyCategories: ['performance'],
     onlyAudits: ['works-offline'],
+    port: 9222,
```

Comment:
This should be a separate example. Can you pull out the sentence you added and make a new `js` block immediately following this one?

Reply:
Yeah happy to do so.